### ISSUE TITLE
Android: Could not find a package configuration file provided by "openssl"

### ISSUE THREAD

**Issue by vince-roy** [OP-Engineering/op-sqlcipher#7]
When trying to create an Android build, the following error is seen:

```
> Task :op-engineering_op-sqlcipher:configureCMakeDebug[arm64-v8a] FAILED
C/C++: CMake Error at CMakeLists.txt:54 (find_package):
C/C++:   Could not find a package configuration file provided by "openssl" with any
C/C++:   of the following names:
C/C++:     opensslConfig.cmake
C/C++:     openssl-config.cmake
C/C++:   Add the installation prefix of "openssl" to CMAKE_PREFIX_PATH or set
C/C++:   "openssl_DIR" to a directory containing one of the above files.  If
C/C++:   "openssl" provides a separate development package or
```

This happens on a fresh Expo install on an M1 Mac running Ventura. 
Here's the repository used to generate the error: https://github.com/vince-roy/op-sqlcipher-android.

**Comment by ospfranco** [comment]
ah, didn't try to run this on Expo, I think they also use open-ssl so they might have some conflicting config

**Comment by ospfranco** [comment]
It seems I botched the openssl dependency. Should be fixed on the latest version `2.0.8`. I tried it to run on the expo project and you will need to add the following on the `android` section in the `app/build.gradle`:

```gradle
packagingOptions {
    excludes = [
        "**/libcrypto.so"
    ]
}
```

Then I tried to compile the app again, but I'm getting other errors not related to this library and I don't want to debug expo. If you can fix them and get the package to run, let me know.

**Comment by vince-roy** [comment]
I can confirm that `2.0.8` fixes the issue on my end. Thank you!
